homeassistant/components/vicare/sensor.py

```diff
@@ -77,6 +77,22 @@ class ViCareSensorEntityDescription(SensorEntityDescription, ViCareRequiredKeysM
         device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    ViCareSensorEntityDescription(
+        key="boiler_supply_temperature",
+        name="Boiler Supply Temperature",
+        native_unit_of_measurement=TEMP_CELSIUS,
+        value_getter=lambda api: api.getBoilerCommonSupplyTemperature(),
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    ViCareSensorEntityDescription(
+        key="hotwater_out_temperature",
+        name="Hot water out Temperature",
+        native_unit_of_measurement=TEMP_CELSIUS,
+        value_getter=lambda api: api.getDomesticHotWaterOutletTemperature(),
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
```

Comment:
Please use consistent capitalisation for the name field

Reply:
I see it now below.  Let me know if there is anything else.  Thanks!

Reply:
I get it but went with how other sensors are currently named.  Anything that had "Temperature" used Title case.   To be certain, should it be "Hot water out temperature"?   And is "Boiler Supply Temperature" ok?  If it should be one standard for all sensors (existing ones) then I can update them all to match.  Just let me know. 

Thanks again

Reply:
Sorry you are right. It changes half way through the original file which is not consistent.

I am not 100% sure what the convention is but I expect title case based on looking around at some other instances briefly.

Reply:
Yes, I have checked a few more.  Title case seems the most common.

**So I Suggest Go With Title Case For Now On The New Code**

Then if you want to change the rest to match, do that as a follow up PR.